### DISCUSSION TITLE
Speed Insights (collecting data) not working

### DISCUSSION THREAD

**Discussion by dvrgoc** [vercel/community#2656] [Help] [answered]
### Summary

Hi everyone,

I have a question so if someone can provide me some info how to resolve this issue.
I would like to enable Speed Insights on a [project](https://www.xbodybysuzana.com.hr/) that I would like to get up and running, but I get the message

- Awaiting Data
- If your project meets the requirements listed below, the first data will be available immediately after the next visit to your project (check for content blockers if you don't see data within 30 seconds)."
- Your project is ready to start collecting Speed Insights.
- (checkbox icon in checked state) Project is ready

All the CSP rules that I am using throw no errors in the console, but the Speed Insights still does not work.
```html
<meta 
      http-equiv="Content-Security-Policy" 
      content="
        default-src 'self'; 
        img-src 'self' data: https://*; 
        child-src 'self' https://www.youtube.com https://www.google.com; 
        style-src 'self';
        style-src-elem 'self' 'unsafe-inline' https://fonts.googleapis.com; 
        font-src 'self' https://fonts.gstatic.com;
        connect-src https://vitals.vercel-insights.com https://google-photos-album-demo2.glitch.me;
        " />
```
Any ideas what am I missing?  Thanks!

### Example

https://www.xbodybysuzana.com.hr/

### Steps to Reproduce

check page source of the URL of the site: https://www.xbodybysuzana.com.hr/ to see the implementation of `<meta />` tag

**Comment by z4nr34l** [comment]
Deployment that you linked, looks like there is no configured Vercel Analytics & Vercel Speed Insights.

Please provide me with access to code so I would be able to help you directly or please follow [instruction in this docs](https://vercel.com/docs/concepts/speed-insights).

**Reply by z4nr34l** [reply]
As I thought now maybe you forget to redeploy project after enabling insights? :D

**Comment by austintoddj** [comment]
@dvrgoc Were you ever able to find an answer to this?

**Reply by dvrgoc** [reply]
Hi @austintoddj 

nope, it is still in the same "stuck" state with disabled "Project is ready" button

**Comment by lure23** [comment]
I have this same, with a SvelteKit project. Deployed at https://app.outstanding.earth

**Comment by enesunal-m** [comment]
I also have the same issue with my SvelteKit project. I think it's a good feature, and I would want to use it if I can. I hope that they fix it very soon.

**Comment by littlepriceonu** [comment]
Same thing happening in my SvelteKit project, sucks to see a good feature go to waste if you use SvelteKit.

**Comment by amyegan** [comment]
I tried to replicate the problem with a fresh copy of the [SvelteKit Boilerplate](https://vercel.com/templates/svelte/sveltekit-boilerplate) template. I deployed and then added Speed Insights without changing any other settings.

When I tried opening the site in a browser where I have very restrictive script blocking settings, I see `vitals | (blocked:other)` in the dev tools Network tab and `POST https://vitals.vercel-analytics.com/v1/vitals net::ERR_BLOCKED_BY_CLIENT` in the dev tools console. This means the no Speed Insights datapoint could be collected and no speed insights data is available to display.

When I tried again in a different browser where I have less restrictive settings, the POST was handled as expected and I was able to see new Speed Insights data in my project dashboard.

**Reply by Martin-Carlsson** [reply]
After following the documentation, I was also unable to install Vercel Speed Insights on Sveltekit.

It looks like the documentation are a bit out of data.

<img width="876" alt="image" src="https://github.com/vercel/community/assets/7769335/b4385fd5-789d-4d7a-ab13-29fb19ef69cd">

**Reply by followdarko** [reply, marked as answer]
I replicated eveything just like in the repo https://vercel.com/templates/svelte/sveltekit-boilerplate
and everything works well! Though i spent couple of hours to figure out what went wrong

**Comment by t1m4lc** [comment]
Same issue with angular :/
